src/app/return-credentials/tds-return/tds-return.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import {SyncupApiService} from 'src/app/shared/api/syncup-api.service';
+import {FormGroup, FormBuilder} from '@angular/forms';
+
+@Component({
+  selector: 'app-tds-return',
+  templateUrl: './tds-return.component.html',
+  styleUrls: ['./tds-return.component.css']
+})
+export class TdsReturnComponent implements OnInit {
+
+  private tdsReturnForm: FormGroup;
+
+  constructor(private formBuilder: FormBuilder, private apiService: SyncupApiService) {
+    this.tdsReturnForm = formBuilder.group({
+                    tdsTanNo: [''],
+                    tdsUserName: [''],
+                    rocPassword: [''],
+                    tracesTdsUserName: [''],
+                    tracesTdsPassword: ['']
+                  });
+   }
+
+  ngOnInit() {
+  }
+
+  private addTdsReturnInfo() {
+    console.log(this.tdsReturnForm.value);
```

Comment:
todo mention kardo ?
//TODO: call backend layer 
something like this

Reply:
Can do